newton/_src/solvers/sph/basic_kernels.py

```
@@ -0,0 +1,285 @@
+    phase_i = sph_phase[i]
+    body_i = sph_body[i]
+    h_i = _support_radius(sph_smoothing_length[i], default_smoothing_length)
+
+    viscosity_i = sph_viscosity[i]
+    if viscosity_i < 0.0:
+        viscosity_i = default_viscosity
+
+    accel = particle_f[i] * inv_mass_i + gravity[wp.max(world_i, 0)]
+    xsph_delta = wp.vec3(0.0)
+    query = wp.hash_grid_query(grid, x_i, h_i)
+    j = int(0)
```

Ruff (0.15.18)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)